apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -818,6 +818,132 @@ public void loadBeneficiaryWhenUpdateAndNon2022EnrollmentDateAndFilterOffExpectR
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample(
+        "non-Bene sample",
+        CcwRifLoadTestUtils.getLoadOptionsWithFilteringofNon2022BenesEnabled(true),
+        stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is on, and a non-Beneficiary type, expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOnExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample(
+        "non-Bene sample",
+        CcwRifLoadTestUtils.getLoadOptionsWithFilteringofNon2022BenesEnabled(false),
+        stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is off, and the LoadStrategy.INSERT_IDEMPOTENT is used with a non-Beneficiary type,
+   * expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOffAndInsertStrategyExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample("non-Bene sample", CcwRifLoadTestUtils.getLoadOptions(), stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is off, and a non-Beneficiary type, expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOffExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample("non-Bene sample", CcwRifLoadTestUtils.getLoadOptions(), stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when UPDATE,
+   * filter setting is off, and a non-Beneficiary type expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenUpdateAndFilterOffExpectRecordLoaded() {
```

Comment:
Largely duplicative with `loadSampleU`.

Reply:
Added a check for ensuring no skips in sampleU tests and removed the duplicative tests for updates.